lms/djangoapps/edraak_university/forms.py

```diff
@@ -0,0 +1,44 @@
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from models import UniversityID
+
+
+class UniversityIDForm(forms.ModelForm):
+    full_name = forms.CharField(
+        label=_('Full Name *'),
+        help_text=_(
+            'Enter your full name that you use at the university (same as your profile page).'
```

Comment:
Recommended change:
- Enter your full name that you use at the university
Remove the brackets and what's within to prevent confusion (students may think: is this a constraint)?